src/components/SingFooter/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { SingFooter } from "../Footer/style";
+import {InboxInput } from "../Footer/style";
+function SingFootersection() {
```

Comment:
```suggestion
function SignInFooter() {
```
and if the footer is the same for the whole app, you can name it just ``Footer``